tests/assembly/manifest.test.ts

```diff
@@ -292,7 +296,9 @@ describe('cloud assembly manifest reader', () => {
 
             expect(() => {
                 AssemblyManifestReader.fromDirectory(path.dirname(manifestFile));
-            }).toThrow(/no such file or directory, open '\/tmp\/foo\/bar\/does\/not\/exist\/MyNestedStack.template.json'/);
+            }).toThrow(
+                /Failed to read CloudFormation template at path: \/tmp\/foo\/bar\/does\/not\/exist\/MyNestedStack.template.json/,
```

Comment:
We were asserting on the runtime error which must be different between node versions as I was getting a different error locally vs in CI. Updated this to assert on the error that we throw.